gulpfile.js

```diff
@@ -0,0 +1,36 @@
+var del = require('del');
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var notify = require("gulp-notify");
+var browserSync = require('browser-sync').create();
+
+gulp.task('sass', function () {
+    return gulp.src('src/breakpoint.scss')
+        .pipe(notify("Compiling SASS!"))
+        .pipe(sass()) 
+        .pipe(gulp.dest('dist/'))
+        .pipe(browserSync.reload({
+            stream: true
+        }))
+});
+
+gulp.task('watch', ['clean', 'browserSync', 'sass'], function () {
+    gulp.watch('src/**/*.scss', ['sass']);
+    gulp.watch('*.html', browserSync.reload);
+});
+
+gulp.task('browserSync', function () {
+    browserSync.init({
+        server: {
+            baseDir: '.'
+        },
+    })
+});
+
+gulp.task('clean', function () {
+    return del.sync('dist');
+});
+
+gulp.task('build', [`clean`, `sass`], function () {
+    console.log('Building files');
```

Comment:
Would it be possible to also add a minifying task as well? so like build the css and min.css?

Reply:
Yes, it is possible. I'll get onto that

Reply:
💯